packages/flutter_test/test/widget_tester_live_device_test.dart

```diff
@@ -82,31 +82,36 @@ No widgets found at Offset(1.0, 1.0).
       ),
     );
 
+    final Size originalSize = tester.binding.createViewConfiguration().size;
```

Comment:
Why not have this call take a surface size and return a new binding to use for the rest of this test? That way there is no state stored based on size.

Reply:
The binding with the new surface size needs to be attached to the WidgetTester so that methods like `tester.pump()` will see the updated binding's state.

You could create a new binding instance and set `tester.binding` to that object.  But you would still need to revert the WidgetTester to the old binding before any other test uses it.